datafusion/physical-plan/src/execution_plan.rs

```diff
@@ -548,10 +563,22 @@ pub trait ExecutionPlan: Debug + DisplayAs + Send + Sync {
     ///   This can be used alongside [`FilterPushdownPropagation::with_filters`] and [`FilterPushdownPropagation::with_updated_node`]
     ///   to dynamically build a result with a mix of supported and unsupported filters.
     ///
+    /// There are two different phases in filter pushdown, which some operators may handle the same and some differently.
```

Comment:
While I love documentation, I would personally suggest not duplicating the docs here as duplicates can get out of sync, and instead leave a link to `FilterPushdownPhase` and focus on getting that documentation to be as clear as possible